### ISSUE TITLE
Missing project contribution guidelines

### ISSUE THREAD

**Issue by mdmundo** [Adventech/sabbath-school-web#8]
Hi guys, I would like to contribute in the web dev app. Where can I start? Provide instructions please.

@nadchif @VitalikL @arashsheyda @hebertialmeida

**Comment by gersonmontenegro** [comment]
Yes, we need it. I sent a PR weeks ago, I didn't get a feedback, or comment. I know this is voluntary, but contributions also are voluntaries, and we want to help. Thanks.

**Comment by VitalikL** [comment]
Sorry @gersonmontenegro I only saw it today.

Thanks @mdmundo for willingness to help. This is a Vue 3 app with tailwind.css framework. If you already have webdev experience it should be straighforward:

1. Checkout the code2. 
2. Install all deps
3. Run it locally (local should point to `stage` env for lesson API) by running `npm run dev`

Do you have any specific ideas you would like to contribute with?
